README.md

```diff
@@ -6,16 +6,16 @@ Dokumentasjonssiden for designsystemet er https://kartverket.github.io/kvib/stor
 
 1. Per dags dato er det ikke klart til å gi flere skrivetilganger direkte i repoet.
 
-2. Alle som vil hjelpe til på KVIB må **forke repoet** og lage pull request.
+2. Alle som vil hjelpe til på KVIB må lage ny branch og åpne en pull request.
 
 3. Pull requesten vil bli sett på i løpet av **1 - 3 virkedager.** De som skal godkjenne pull requestene er:
```

Comment:
Mitt forslag er å fjerne alt dette (linje 7-11) og heller flytte linje 21 opp på toppen😎 "Sjekk ut [linken] for å bidra til designsystemet"